Add SetProfile to API

shughes-uk opened the pull request.

shughes-uk committed:
Add SetProfile

shughes-uk committed:
Marshall onto main thread from plugin thread

notr1ch commented:
There seem to be mixed spaces / tabs in these commits. Please check your tabs are set to soft-tabs (4 spaces).

shughes-uk committed:
convert some tabs to spaces

shughes-uk committed:
Merge branch 'master' of https://github.com/jp9000/OBS.git

shughes-uk committed:
fix more tabs

shughes-uk commented:
Forgot to set my settings in VS! Should be fixed now, caught some other ones too.

notr1ch committed:
Don't load GIF files that overflow 32 bit pointers

notr1ch committed:
Fix whitespace

Commit:
services: update Hitbox ingest servers list

Modified: Asia

notr1ch committed:
Always flush encoder output if using QSV / NVENC

NVENC at least does not like being shut down if there is still pending encoder output to be consumed. Normally output flushing is skipped when rendering preview for UI responsiveness.

Gol-D-Ace committed:
Update twitch.tv ingest server list

SapiensIsRad committed:
Update YouTube recommended keyint

The recommended keyint for YouTube live streaming is 4 seconds -
https://support.google.com/youtube/answer/2853702

CommanderRoot committed:
Update YouTube recommended encoding setting

Youtube recommends to use cbr encoding:
https://support.google.com/youtube/answer/2853702

Gol-D-Ace committed:
Update twitch.tv ingest server list

notr1ch committed:
Additional .exe check for Witcher 3 (GoG version)

notr1ch committed:
Support adding folders to the Image Slideshow

The contents of the folder will be loaded on each source initialization, allowing files to be added / removed without having to update the source.

Glought committed:
Confirm exit if exiting while streaming/recording

If OBS is live/recording when the user presses the Exit, File/Exit, or
the [X] Button a message box will open asking if the user wants to exit
OBS while live/recording.

This is useful to protect against accidentally closing OBS while
streaming/recording.

CLoses jp9000/OBS#448

jp9000 committed:
Set version to 0.652

Gol-D-Ace committed:
Add latest translations from CrowdIn

Gol-D-Ace committed:
Update authors file with new translators

Gol-D-Ace committed:
Update twitch.tv ingests

Gol-D-Ace committed:
Update twitch.tv ingests

notr1ch committed:
Fix image slideshow folder support not functioning

jp9000 committed:
Update version to 0.653

jp9000 committed:
Update D3D9 patches for windows 10

jp9000 committed:
Update version to 0.654b

jp9000 committed:
Update Twitch ingests

jp9000 committed:
Get windows version via kernel32.dll

Microsoft basically deprecated GetVersion/GetVersionEx so now they'll
always report invalid version numbers.  The way to get the actual
version of windows after 8+ is to get the actual file version of
kernel32.dll according to microsoft's documentation.

jp9000 committed:
Fix incorrect function typedefs

jp9000 committed:
Update assembly file to indicate win 10 support

jp9000 committed:
Update D3D9 exceptions for win 10 update

Performance fix for D3D9 version 10.0.10240.16412

jp9000 committed:
Update version to 0.655

JIEgOKOJI committed:
Update goodgame recommended encoding setting

Update goodgame recommended encoding setting
http://goodgame.ru/help/2/16/

ThoNohT committed:
Don't snap crop when resizing a source

When resizing a source with snapping enabled, any crop that was small enough to
be snapped would be reset as if the user was dragging the crop line with snap
enabled. The crop should not be affected by a resize action (unless the source
becomes too small), so disable the snap check on crops when resizing.

ThoNohT committed:
Don't try to deselect when clicking on nothing

When after transforming a source, the user tries to ctrl + click on nothing,
OBS would try to deselect whatever source was there, which resulted in an
assertion failure. This adds a check that there has to be something to deselect.

Gol-D-Ace committed:
Add latest translations from Crowdin

Gol-D-Ace committed:
Update authors file with new translators

notr1ch committed:
Add support for signature checking updates

Both the manifest and updater.exe must now be signed before they will be used. Uses a hard coded RSA public key.

notr1ch committed:
Save scenes / sources on Windows shutdown / restart.

notr1ch committed:
Check AppData permissions and disk space on startup.

Gol-D-Ace committed:
Add YouTube Gaming reference

YouTube Gaming is live since today (26 August 2015) and people will ask
for it.
This makes it a bit clearer that YouTube and YouTube Gaming
(which share the same ingestion system) work with OBS.

Commit:
rtmp-services: Update Hitbox servers list

Added: EU-East

palana committed:
Add more QSV CPUIDs

palana committed:
Assume QSV being supported means Haswell or better

This should work for Broadwell/Skylake/whatever comes next, could
potentially break for new chips with old Quick Sync units though

palana committed:
Add missing libs in x64 build

This was broken in "Add support for signature checking updates"
(2e431182481d4869fa6c2a3abc52dcf6557265e7)

notr1ch committed:
FLVFileStream: Fix incorrect tag lengths

http://trac.ffmpeg.org/ticket/4867

notr1ch committed:
Installer: Update DirectX setup link

Microsoft replaced the download on their site with a Windows 10 installer tool for some bizarre reason.

jp9000 committed:
Update to version 0.656

Gol-D-Ace committed:
Add latest translations from Crowdin

Gol-D-Ace committed:
Update authors file with new translators

Gol-D-Ace committed:
Remove Pirate English locale files

nmlgc committed:
Include the terminating \0 character in string measurements of text sources

Apparently, \0 is not always a zero-width character?
Not including it as part of the string measurement may cut off the final word
on the line when using certain fonts together with outlines and wrapping.

Test case:
• Font: M+ 2p black (downloadable at https://mplus-fonts.osdn.jp/mplus-outline-fonts/download/index-en.html#download, "M+ TESTFLIGHT 060" release)
• ☑ Use Outline
• ☑ Use Custom Text Extents
• ☑ Wrap
• ☐ Scroll Mode (doesn't seem to be affected)
• Text: A, 20

notr1ch committed:
Installer: Remove 3rd party XINPUT download link

The 3rd party site has very shady ads / malware or is hacked.

jp9000 committed:
Fix whitespace violation

notr1ch committed:
Small fixes from Coverity Scan

Gol-D-Ace committed:
Update Twitch ingests

palana committed:
Add QSV Sandy Bridge + D3D11 profile

Gol-D-Ace committed:
Update Vaughn Live / iNSTAGIB.tv ingests

Gol-D-Ace committed:
Update Twitch ingests
